pkg/cloud/aliyunprovider.go

```diff
@@ -782,24 +791,11 @@ func (alibaba *Alibaba) GetKey(mapValue map[string]string, node *v1.Node) Key {
 		client, err = sdk.NewClientWithAccessKey(slimK8sNode.RegionID, aak.AccessKeyId, aak.AccessKeySecret)
```

Comment:
@nikovacevic , the panic was here, before the code fix i was setting `skipSystemDiskRetrieval` to true and then code would follow into create a sdk client with the keys `aak.AccessKeyId` `aak.AccessKeySecret` expected to be initialized but where nil...

Reply:
So `alibaba.accessKey` was `nil`? And you were trying to access `alibaba.accessKey.AccessKeyId` and `alibaba.accessKey.Secret`? If so, I would recommend a manual `nil` check there (`if aak == nil`), even if your change here appears to solve it. Best to handle such things explicitly.

Reply:
Thanks @nikovacevic . I have added this needed condition.